vo-dml/mango.vo-dml.xml

```diff
@@ -904,58 +890,62 @@ As long as the vocabulary is not set, the possible values of this attribute are
     <vodml-id>EpochPositionErrors</vodml-id>
     <name>EpochPositionErrors</name>
```

Comment:
The migration from composition to attribute includes a change to the multiplicity of the different errors.  This now requires errors for all properties where before they were not.  Is that intentional?